contracts/FailingOracle.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: GPL-3.0-only
+// solhint-disable no-empty-blocks
+pragma solidity ^0.7.5;
+
+import "./PriceOracle.sol";
+
+contract FailingOracle is PriceOracle {
+    constructor() {}
+
+    function updatePrices() public override {}
+
+    function consultUsdEth(uint256 _usdAmount) public view override returns (uint256) {
+        revert("FailingOracle::consultUsdEth: this oracle always fails");
```

Comment:
This message is correct, but useless and confusing for the the end user. We're using this oracle in one place anyway and for one purpose, how about:
```suggestion
        revert("ETH not supported yet, please use RAD");
```